security/index.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: news
+title: Security Vulnerabilities
+filename: security
+main-blurb: Known vulnerabilities, workarounds, and resolutions
+---                      
+        <!-- begin Security -->
+        <hr class="whitespace">
+        <div id="community-posts" class="row">
+
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-code-fork"></i> OMERO versions 5.2.7 and earlier</h6>
+                <h2><a href="{{ site.baseurl }}/security/2017-sv1-filename-mutability.html">2017-SV1 Filename Mutability</a></h2>
+            </div>
+            <hr class="medium-8">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-code-fork"></i> OMERO versions 5.2.7 and earlier</h6>
+                <h2><a href="#">2017-SV2 Edit in RW Group</a></h2>
```

Comment:
This filler # is breaking Travis:
- ./_site/www.openmicroscopy.org/security/index.html
  *  linking to internal hash # that does not exist (line 91)
     <a href="#">2017-SV2 Edit in RW Group</a>
htmlproofer 3.7.2 | Error:  HTML-Proofer found 1 failure!